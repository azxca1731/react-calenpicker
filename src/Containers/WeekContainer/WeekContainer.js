import React from "react";
import PropTypes from "prop-types";

import { CssConnector } from "Containers/Provider";
import Week from "Components/Week";

const WeekContainer = props => {
  return <Week {...props} />;
};

WeekContainer.propTypes = {
  weekNumber: PropTypes.oneOf([0, 1, 2, 3, 4, 5]).isRequired,
  cssObject: PropTypes.object
};

export default CssConnector(({ state }) => ({
  cssObject: state.WeekCssObject
}))(WeekContainer);
