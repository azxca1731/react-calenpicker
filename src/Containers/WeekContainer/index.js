import WeekContainer from "./WeekContainer";

export default WeekContainer;
