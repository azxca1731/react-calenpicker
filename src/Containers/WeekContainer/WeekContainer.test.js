import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import WeekContainer from "./index";

describe("WeekDay", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<WeekContainer />);
    expect(wrapper).toHaveLength(1);
  });

  it("renders with snapshot", () => {
    const wrapper = shallow(<WeekContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
