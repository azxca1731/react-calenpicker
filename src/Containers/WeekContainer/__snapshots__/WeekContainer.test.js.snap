// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekDay renders with snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
